exercise2/assets/style2.css

```diff
@@ -0,0 +1,77 @@
+
+
+
+.contimg{
+	width: 30%;
+	height: 10%;
+	margin: 1.5%;
```

Comment:
Margins normally are not used with percentages, is better to use `px` for this case.